sympy/polys/specialpolys.py

```diff
@@ -87,7 +87,10 @@ def cyclotomic_poly(n, x=None, polys=False):
 
 @public
 def symmetric_poly(n, *gens, **args):
-    """Generates symmetric polynomial of order `n`. """
+    """Generates symmetric polynomial of order `n`.Returns
```

Comment:
This should be formatted differently. From [PEP 257](https://www.python.org/dev/peps/pep-0257/#id17):

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.